Google Play Music.sln

```diff
@@ -17,8 +17,8 @@ Global
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{B2C52D42-4EEE-445C-AD6D-357D93397937}.Debug|Any CPU.ActiveCfg = Release|x86
```

Comment:
That's understandable. For what it's worth, I changed the configuration to Debug so that code optimization wouldn't prevent full debug sessions. I'll flip it back if you'd like.